### DISCUSSION TITLE
Playtime: Immediate "Next Loop Record" after setting tempo with new recording from silence mode

### DISCUSSION THREAD

**Discussion by RibbleFripFnibbulator** [helgoboss/helgobox#1281] [Ideas]
1. Silence mode
2. Record into a blank slot (exclusive mode disabled)
3. ????
4. Tempo is set, first clip plays back, and the clip below it is already recording at the same tempo immediately on FIRST playback

So far with all my methods I am not able to achieve this. I put together a Trigger->Browse down->Trigger script hoping that maybe if I send the commands quickly enough in succession, the second loop trigger's lateness would be forgiven and it would still enter record mode immediately. However it appears this is not possible. Playtime will wait until one "cycle" passes before the second loop begins to record. 

Can this be implemented? An immediate "next layer overdub" if you will? This would allow Playtime to be used for faster live-looping.

**Comment by helgoboss** [comment]
Could you please add this to #1005 for now? This would be another recording mode. Maybe it's mentioned there already.

There's no way to achieve this cleanly with current means.
